app/views/statements/list.html.haml

```diff
@@ -1,4 +1,4 @@
-- # This template appears to be unused.  If nothing is logged to rollbar by June 2023, delete this file.
+- # TO DO: This template appears to be unused.  If nothing is logged to rollbar by June 2023, delete this file.
```

Comment:
Why adding "TO DO" instead of deleting the file?

Reply:
Good question :) I checked rollbar and confirmed there are no warnings here, so I'll go ahead and remove it 🗑️